.changeset/tasty-moose-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/backend-jsonrpc": minor
```

Comment:
We usually don't bump version of the backend-jsonrpc package for adding linter rules.

```suggestion
```